Documentation/README.md

```diff
@@ -37,16 +37,19 @@ For uniqueness, the filename of a document in the BCF must be the document guid.
 
 ### Topic folder structure inside a BCFzip archive
 
-The folder name is the GUID of the topic. This GUID is in the UUID form. The GUID must be all-lowercase. The folder contains the following files:
+The folder name is the GUID of the topic. This GUID is in the UUID form. The GUID must be all-lowercase. The folder contains the following file:
 
 * markup.bcf
     * An XML file following the markup.xsd schema that is described below.
-* viewpoint.bcfv
+
+Additionally the folder can contain pair of files:
```

Comment:
Should this be:
```
Additionally the folder can contain pairs of viewpoint and snapshot files:
```

Reply:
Or 
```
Additionally the folder can contain viewpoint and/or snapshot files:


```

Reply:
well, do not necessary have to be a pair. And also, it can contain bitmap files